### PR TITLE
doc: add module diligence section to README

### PR DESCRIPTION
**Pull request by BethGriggs**
Opening a Draft PR just to start collating feedback on the minimum diligence we should be doing on each of the recommended modules. I've added just some personal ideas just to start. Some of them may seem obvious, but I thought it would be good to explicitly call them out in case it is something we can easily automate as a check.

I propose we spend 10 minutes discussing/defining/expanding this criteria as a group in our next meeting. Following that discussion/agreement, we can explore automating some of these checks (I know there are existing tools for checking licenses, etc.).

Also, I just added this to the README for now, but I am not sure that is the best place for it to live.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,17 @@ with only a subset of sections having recommendations):
   * Load-balancing
   * [Failure Handling](./docs/operations/failurehandling.md)
 
+## Module Diligence
+
+As a minium, any modules recommended as part of this reference architeucture should meet the following criteria:
+
+* The module is not deprecated.
+* The module has an appropriate license.
+  * _Can we define what we consider an 'appropriate' license is in this context?_
```
Comment (mhdawson) [inline README.md:92]: I think we could define a set of licences that are generally acceptable (MIT, Apache 2 etc.) and those that are not in the list should be flagged for additional review/agreement that it still makes sense to recommend the module.

**Comment by mhdawson** [comment]
Looks like a good starting list to me.

**Comment by lholmquist** [comment]
Looks good

**Comment by mhdawson** [comment]
When we discuss we should talk about where including `npm audit is clean` makes sense to include.

**Review by mhdawson — COMMENTED** [review]
LGTM

**Review by lholmquist — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -83,6 +83,20 @@ with only a subset of sections having recommendations):
   * Load-balancing
   * [Failure Handling](./docs/operations/failurehandling.md)
 
+## Module Diligence
+
+As a minium, any modules recommended as part of this reference architeucture should meet the following criteria:
```
Comment (BethGriggs) [inline README.md:88]:
```suggestion
As a minimum, any modules recommended as part of this reference architecture should meet the following criteria:
```
fixup typos

**Review by AshCripps — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**